test/js/bun/shell/env.positionals.test.ts

```diff
@@ -9,85 +9,85 @@ $.nothrow();
 describe("$ argv", async () => {
   for (let i = 0; i < process.argv.length; i++) {
     const element = process.argv[i];
-    TestBuilder.command`echo $${i}`
+    TestBuilder.command`echo $${{ raw: i }}`
       .exitCode(0)
       .stdout(process.argv[i] + "\n")
       .runAsTest(`$${i} should equal process.argv[${i}]`);
   }
 });
 
-test("$ argv: standalone", async () => {
-  const script = path.join(import.meta.dir, "fixtures", "positionals.bun.sh");
-  const { stdout, stderr, exited } = spawn({
-    cmd: [bunExe(), "run", script, "a", "b", "c"],
-    stdout: "pipe",
-    stdin: "ignore",
-    stderr: "pipe",
-    env: bunEnv,
-  });
+// test("$ argv: standalone", async () => {
```

Comment:
why are these commented out?

Reply:
This was commented out for debugging some things, uncommented them